docs/.vitepress/config.js

```diff
@@ -0,0 +1,128 @@
+export default {
+  title: " ",
+  description: "Privacy First eSigning",
+  cleanUrls: true,
+  lastUpdated: true,
+
+  socialLinks: [
+    { icon: "github", link: "https://github.com/certifaction/cli" },
+  ],
+
+  editLink: {
+    pattern: "https://github.com/certifaction/cli/edit/main/docs/:path",
+    text: "Edit this page on GitHub",
+  },
+
+  footer: {
+    copyright: "Copyright © 2020-present Certifaction AG",
+  },
+
+  themeConfig: {
+    siteTitle: false,
+    logo: "/certifaction_logo.svg",
+    nav: [
+      {
+        text: "Guide",
+        link: "/guides/what-is-certifaction",
+        activeMatch: "/guides/",
+      },
+      {
+        text: "Reference",
+        link: "/references/api",
+        activeMatch: "/references/",
+      },
+      {
+        text: "Changelog",
+        link: "https://github.com/certifaction/cli/releases",
+      },
+    ],
+    sidebar: {
+      // This sidebar gets displayed when a user
+      // is on `guide` directory.
+      "/guides/": [
+        {
+          text: "Introduction",
+          collapsed: false,
+          items: [
+            {
+              text: "What is Certifaction?",
+              link: "/guides/what-is-certifaction",
+            },
+            { text: "Getting started", link: "/guides/getting-started" },
+            {
+              text: "Document signature",
+              collapsed: false,
+              items: [
+                {
+                  text: "Digital signature",
+                  link: "/guides/digital-signature",
+                },
+                {
+                  text: "Signature request",
+                  link: "/guides/signature-request",
+                },
+                {
+                  text: "Digital Twin",
+                  link: "/guides/digital-twin",
+                },
+                {
+                  text: "Local Processing",
+                  link: "/guides/local-processing",
```

Comment:
The introduction section shares the same path (/guides/) as the actual guides. Can we use another path for introduction? I suggest /concepts/

Reply:
This is done so that it appears in the same left sidebar.  We can separate how it looks and where the file is stored.  I have followed the structure of the VitePress documentation, using collapsable sections.  If you change the path prefix, you force having an other side bar, like between Guide and Reference.  Having several tiny sidebars looks weirds IMHO and I rather have collapsable.